src/ui/app/package-lock.json

```diff
@@ -1,9411 +0,0 @@
-{
```

Comment:
Was this intentionally removed @agiron123?

Reply:
I took this out to get fresh packages, but I expected it to be re-generated again for me. When installing via npm link apparently this does not get generated.

I will go ahead and publish the common package and re-generate this file before merging.

Reply:
So there was some weirdness with the package-lock being regenerated properly and what not. 

Instead there is another package to use, according to this github thread:
https://github.com/rogeriochaves/npm-force-resolutions/issues/36

Things are now working properly and this package is now used in both common and app.